_posts/2022-08-11-Agroserv.md

```diff
@@ -0,0 +1,17 @@
+---
+site: [pasteur, freiburg, erasmusmc, elixir-it, belgium, genouest]
+title: We are part of the Horizon Europe project AgroServ
+tags:
+author: aerxleben
+supporters:
+---
+
+**We are pleased to be part of Horizon Europe funded project consortia AgroServ**
+
+AgroServ supports research and innovation by providing customised and integrated research infrastructure services in all fields related to this challenge, spanning from molecular to organism, to ecosystem, to communities and society. It takes place in line with the One-Health approach, with particular regard to threats and risks on agroecosystems and to enhance new agro-ecological practices and their socio-economic benefits. 
+
+AgroServ will have a high impact on the future of the food system, preserving biodiversity and reducing the impact of agriculture on climate. The consortium will work closely with partners from the society, farmers, industry, citizens and policy makers, through living labs and towards the establishment of evidence-based policy, and co-developed practices in agriculture.
+
+The Freiburg Galaxy Team will join AgroServ and provides the Galaxy infrastructure and support, user support, and training.
+
+This five-years project has a budget of 15 million euros funded via Horizon Europe. Project kick-off is in October 2022.
```

Comment:
maybe move this sentence up where the project is described? And finish the post saying that Freiburg participates